kirovy/objects/__init__.py

```diff
@@ -0,0 +1,45 @@
+import datetime
+from dataclasses import dataclass, field, fields, Field
+
+from kirovy import typing as t
+
+
+@dataclass
+class CncnetUserInfo:
+    id: int = field()
+    name: str = field()
+    email: str = field()
+    created_at: datetime.datetime = field()
+    updated_at: datetime.datetime = field()
+    group: str = field()
+    ip_address_id: int = field()
+    email_verified: bool = field()
+    avatar_path: t.Optional[str] = field()
+    avatar_upload_allowed: bool = field()
+    discord_profile: str = field()
+    youtube_profile: str = field()
+    twitch_profile: str = field()
+    alias: str = field()
+
+    def __init__(self, **kwargs):
+        for user_field in fields(self):
+            value = kwargs.get(user_field.name)
+            if value is None:
+                setattr(self, user_field.name, None)
+            else:
+                self._convert_and_set(value, user_field)
+
+    def _convert_and_set(self, value: t.Any, user_field: Field) -> None:
+        """Convert any relevant types, and set the class attrs.
+
+        :param value:
+            The value from CncNet.
```

Comment:
```suggestion
            The value from CnCNet.
```

Reply:
Acronyms are usually `CncNet` to avoid like `APIUser`. Easy to tell when it's something like API, but with non-standard acronyms it becomes annoying.